src/controller/toolbox_controller.js

```diff
@@ -108,6 +108,23 @@ class ToolboxController {
   }
 
   /**
+   * Adds separator to current Toolbox.
```

Comment:
Looking at what this does in the production dev tools, let's rename this addCategorySeparator() (and update the doc).

FYI: You can also have separators between blocks within a toolbox category flyout.  Added issue #81 to track this issue (easy stretch goal).